dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/condition.py

```diff
@@ -182,4 +201,5 @@ def task_params(self, camel_attr: bool = True, custom_attr: set = None) -> Dict:
         """
         params = super().task_params
         params["dependence"] = self.condition.get_define()
+        params["conditionResult"] = self.condition_result
```

Comment:
Is this line redundant? beause `condition_result ` defined in custom_attr of `Task.task_params`

Reply:
Cause we can not set property `property` in class `Conditions` So I overwrite function `task_params`. It related your above comment https://github.com/apache/dolphinscheduler/pull/7783/files/2dd5134de92cca727514907e1863ab1db523055c#r778015716 .

Reply:
I would try whether it work when I add setter in class `Task` attribute `condition_result`

Reply:
I add property condition_result getter and setter in 169dd13, PTAL, thanks